lib/awful/hotkeys_popup/widget.lua

```diff
@@ -290,8 +290,8 @@ function widget.new()
                     joined_labels = joined_labels .. rendered_hotkey .. (i~=#_keys and "\n" or "")
                     end
                 current_column.layout:add(wibox.widget.textbox(joined_labels))
-                local max_width = compute_textbox_width(wibox.widget.textbox(max_label_content), s) +
-                                                    widget_instance.group_margin
+                local max_width, _ = wibox.widget.textbox(max_label_content):get_preferred_size(s)
+                max_width = max_width + widget_instance.group_margin
                 if not current_column.max_width or max_width > current_column.max_width then
```

Comment:
This part of the change seems unnecessary? The file can just be left as-is. (Also, I think luacheck will now complain that `compute_textbox_width` is set, but unused)